### PR TITLE
Adds Core Team document

### PR DESCRIPTION
**Pull request by ashfurrow**
This is a draft of the Core Team document discussed in #14. I've tried to lay out the responsibilities and process without being too in-depth. 

This PR is just a draft and is open for discussion from anyone. It's also a living document and will likely evolve over time.

Fixes https://github.com/Moya/contributors/issues/14

### CODE REVIEW

**Comment by ashfurrow** [comment]
I've explicitly not included Code of Conduct enforcement on the Core Team, since we have an existing system for people to contact me directly about CoC issues and I don't mind maintaining that responsibility.

File: Core Team.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Core Team Responsibilities
+
+The Moya Core Team has the following responsibilities:
+
+- Being available to answer high-level questions about Moya's vision and future.
+- Being available to review longstanding/forgotten pull requests.
+- Occasionally check Moya issues, offer input, and categorize with GitHub issue labels.
+- Looking out for up-and-coming members of the Moya community who might want to serve as Core Team members.
+
+Note that the Moya Core Team – and all Moya contributors – are open source _volunteers_; membership on the Core Team is expressly _not_ an obligation. The Core Team is distinguished as leaders in the community and while they are a good group to turn to when someone needs an answer to a question, they are still volunteering their time, and may not be available to help immediately.
```
Comment (orta) [inline Core Team.md:14]: 👍

**Review by orta — APPROVED** [review]

**Review by BasThomas — APPROVED** [review]
Awesome, thanks @ashfurrow!

**Review by SD10 — APPROVED** [review]
@ashfurrow How about adding yourself and @orta? Your involvement is pretty clear to people familiar with Moya but it could be good to clarify for newcomers.

**Review by pedrovereza — APPROVED** [review]
Thanks, Ash!

**Comment by ashfurrow** [comment]
Yup yup – I've added Orta and myself, good catch. 

I'm going to give this another day or so before merging 👍

**Review by colinta — APPROVED** [review]
👍

**Review by AndrewSB — APPROVED** [review]

**Review by pmairoldi — APPROVED** [review]

File: Core Team.md (hunk 1 above)
Comment (freak4pc) [reply Core Team.md:14]: 😍

**Pull request merged**